### PR TITLE
entproto: support field.Strings

### PR DESCRIPTION
**Pull request by rotemtam**
Closes https://github.com/ent/ent/issues/2474

### CODE REVIEW

File: entproto/adapter.go [hunk 1]
```diff
@@ -510,9 +515,20 @@ func extractProtoTypeDetails(f *gen.Field) (fieldType, error) {
 	}, nil
 }
 
+func extractJSONDetails(f *gen.Field) (fieldType, error) {
+	if f.Type.Ident == "[]string" {
```
Comment (giautm) [inline entproto/adapter.go:519]: We also have []float64 and []int

Comment (rotemtam) [reply entproto/adapter.go:519]: Yep, this PR adds support for field.Strings, the rest will come after

File: entproto/internal/entprototest/adapter_test.go [hunk 2]
```diff
@@ -94,6 +94,14 @@ func (suite *AdapterTestSuite) TestImplicitSkippedMessage() {
 	suite.EqualError(err, entproto.ErrSchemaSkipped.Error())
 }
 
+func (suite *AdapterTestSuite) TestMessageWithStrings() {
```
Comment (a8m) [inline entproto/internal/entprototest/adapter_test.go:97]: Do you mind changing the receiver to be a single character? In this PR or a different one. Thanks 🙏

**Review by a8m — APPROVED** [review]

File: entproto/internal/entprototest/adapter_test.go (hunk 2 above)
Comment (rotemtam) [reply entproto/internal/entprototest/adapter_test.go:97]: Sure. I want to deprecate the usage of suite anyways

**Pull request merged**